cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,57 @@
+/// <reference types='cypress' />
+const faker = require('faker');
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('/');
   });
 
-  it('should have the ability to assert automatically resolved alerts', () => {
+  it('should be able to assert alerts that are resolved automatically', () => {
+    cy.get('#alertButton').click();
 
+    cy.on('window:alert', (alertText) => {
+      expect(alertText).to.equal('You clicked a button');
+    });
   });
 
-  it('should have the ability to assert scheduled allert', () => {
+  it('should be able to assert alerts that are scheduled', () => {
+    cy.get('#timerAlertButton').click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(5000);
```

Comment:
Using `cy.wait()` with a hardcoded delay is not recommended in Cypress, as it can lead to flaky tests. Instead, you should use assertions to wait for certain conditions to be met. In this case, you could potentially use `cy.clock()` and `cy.tick()` to control the JavaScript clock, allowing you to fast-forward time and avoid the `cy.wait()`.